module-utils/log/LoggerBufferContainer.hpp

```diff
@@ -42,7 +42,7 @@ class LoggerBufferContainer
     }
 
   private:
-    static constexpr size_t circularBufferSize = 1024;
+    static constexpr size_t circularBufferSize = 512;
```

Comment:
Is this change for both products?

Reply:
For now, we can change this

Reply:
How about we do this only for Harmony by defining `PRODUCT_${PRODUCT}` here:
https://github.com/mudita/MuditaOS/blob/master/CMakeLists.txt#L86
and then checking the `#ifdef`?